config/clusters/callysto/common.values.yaml

```diff
@@ -32,6 +32,28 @@ jupyterhub:
     image:
       name: callysto/2i2c
       tag: 0.1.0
+    extraFiles:
+      tree.html:
```

Comment:
Normally, extending only the `page.html` template would have been enough since both the `tree` and the `notebook` templates are extending it themselves. But for some reason, it turned impossible to make it to work like this... I fear this is because of NotebookApp -> ServerApp? But I have tried everything with no luck, so this is the best version that's actually doing the job